dynamo-formatters/src/main/scala/uk/gov/nationalarchives/dynamoformatters/DynamoFormatters.scala

```diff
@@ -83,6 +84,7 @@ object DynamoFormatters {
   val ingestedPreservica = "ingested_PS"
   val childCount = "childCount"
   val skipIngest = "skipIngest"
+  val location = "location"
```

Comment:
The location has `s3://bucket/key` in it so that should be clear. It's also not inconceivable that this might be run on a file system for very large files or something like that in the future so I'll leave it.